seqr/views/apis/staff_api_tests.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+import json
+import mock
+
+ES_CAT_ALLOCATION=[{
+    u'node': u'node-1',
+    u'disk.used': u'67.2gb',
+    u'disk.avail': u'188.6gb',
+    u'disk.percent': u'26'
+  },
+  {u'node': u'UNASSIGNED',
+   u'disk.used': None,
+   u'disk.avail': None,
+   u'disk.percent': None
+  }]
+
+ES_CAT_INDICES=[{
+    "index": "test_index",
+    "docs.count": "122674997",
+    "store.size": "14.9gb",
+    "creation.date.string": "2019-11-04T19:33:47.522Z"
+  },
+  {
+    "index": "test_index_second",
+    "docs.count": "672312",
+    "store.size": "233.4mb",
+    "creation.date.string": "2019-10-03T19:53:53.846Z"
+  },
+  {
+    "index": "test_index_old",
+    "docs.count": "672312",
+    "store.size": "233.4mb",
+    "creation.date.string": "2019-10-03T19:53:53.846Z"
+  },
+  {
+    "index": "1kg.vcf.gz",
+    "docs.count": "672312",
+    "store.size": "233.4mb",
+    "creation.date.string": "2019-10-03T19:53:53.846Z"
+  }]
+
+ES_CAT_ALIAS=[
+  {
+    "alias": "010203d4c1452eef3f8725bfdee23476",
+    "index": "test_index"
+  },
+  {
+    "alias": "1531a081753f4a135eef25b3496316dd",
+    "index": "1kg.vcf.gz"
+  }]
+
+TEST_INDEX_SRC_PATH="test_index_src_path"
+TEST_INDEX_SECOND_SRC_PATH="test_index_second_src_path"
+TEST_INDEX_OLD_SRC_PATH="test_index_old_src_path"
+TEST_INDEX_1KG_SRC_PATH="test_index_1kg_src_path"
+ES_INDEX_MAPPING={
+  "test_index": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "25",
+          "genomeVersion": "38",
+          "sampleType_1": "WES",
```

Comment:
seqr doesn't care about the `_1` fields - please take them out of the test data

Reply:
Okay, I will delete them. They are actually from the response of the productional system.

Reply:
yes but as I said during our convesation, I would like you to use a minimal subset of fields so you can clearly test everything. Having too much unneccessary test data makes test unreadable